test/testcafe/framework/page-objects/components/BaseFormObjectV3.js

```diff
@@ -0,0 +1,16 @@
+import BaseFormObject from './BaseFormObject';
+import { Selector } from 'testcafe';
+
+const FORM_SELECTOR = '[data-se="form"]';
+const FORM_INFOBOX_ERROR_SELECTOR = '[data-se="message"] [data-se="infobox-error"]';
+
+export default class BaseFormObjectV3 extends BaseFormObject {
+  constructor(t, index = 0) {
+    super(t);
+    this.el = Selector(FORM_SELECTOR).nth(index);
```

Comment:
is this `nth` fn considered an anti-pattern?

Reply:
I pulled this from @lesterchoi-okta PR https://github.com/okta/okta-signin-widget/pull/2648/files#diff-9b67ece18bbf878941e07ad5952c5de98abcf533d0e4bdde0ca8f58ba57d13c3R11

Reply:
even if this isn't the ideal pattern, let's go with the quickest selector pattern if we spend any more time on spec conversion. we will need to return after beta to convert more specs and spend the time then to be more thoughtful

Reply:
is this line needed in this PR, not see any page object initialized with index